### PR TITLE
Major changes

### PR DESCRIPTION
**Pull request by chartjes**
These are a bunch of changes that reflect I want to take the tests in this repo in a much different direction from how they currently exist

### CODE REVIEW

**Comment by rpappalax** [comment]
@chartjes looks some long overdue cleanup!  i see also that you're removing the bash scripts which is fine for the long run for getting things automated, but if i needed to do a deploy in your absence, how would i run those tests?  could you instead temporarily put them into a /scripts subdir until there was automation to replace them?

**Comment by chartjes** [comment]
@rpappalax The only thing that has changed since you were the primary for testing autopush is that there are only two tests in this repo that are used: one that verifies some API responses are as expected and that we can generate an error on Sentry.

Nothing else has changed. The tickets I create for deployment testing list all the steps I am currently doing, and none of them need anything out of this repo.

**Comment by chartjes** [comment]
@rpappalax The only thing that has changed since you were the primary for testing autopush is that there are only two tests in this repo that are used: one that verifies some API responses are as expected and that we can generate an error on Sentry.

Nothing else has changed. The tickets I create for deployment testing list all the steps I am currently doing, and none of them need anything out of this repo.

File: README.md [hunk 1]
```diff
@@ -17,46 +17,38 @@ This repository contains integration tests used largely for deployment verificat
...
-* make docker-tag
-* make docker-push
+Run Tests
+---------
 
+To run the current set of tests, please use the following command:
 
-Jenkins
--------
+`pytest -v --env=<ENV> --api-version=<API_VERSION> tests/`
```
Comment (m8ttyB) [inline README.md]: Nice +1

File: .travis.yaml [hunk 2]
```diff
@@ -8,3 +8,4 @@ script:
   - tox
 notifications:
 email: rpappalax@gmail.com
```
Comment (rpappalax) [inline .travis.yaml]: probably want to change this either to your own email (at first), remove or add fte-ci email if you wish to track the travis runs

File: .travis.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+language: python
+python: 2.7, 3.5
+env:
+  - TOXENV=flake8
+install:
+  - pip install tox
+script:
+  - tox
+notifications:
+email: rpappalax@gmail.com
```
Comment (m8ttyB) [inline .travis.yaml]: w/ chartjes taking over the project, should notifications be tweaked to represent this?

File: .travis.yaml [hunk 4]
```diff
@@ -8,3 +8,4 @@ script:
   - tox
 notifications:
 email: rpappalax@gmail.com 
+
```
Comment (rpappalax) [inline .travis.yaml]: Eventually a Dockerfile will be needed here, but it's OK to remove this one if you wish.  Actually, I have one from Benny and the sync tests that will allow desktop selenium testing that may be a better starting place for this.

File: tests/test_url_checks.py [hunk 5]
```diff
@@ -1,58 +0,0 @@
-import pytest
```
Comment (rpappalax) [inline tests/test_url_checks.py]: is this test being removed?

File: .travis.yaml (hunk 4 above)
Comment (chartjes) [reply .travis.yaml]: I'll revisit the whole Docker issue later on when I have a better plan in place for what I want to do

File: tests/test_url_checks.py (hunk 5 above)
Comment (chartjes) [reply tests/test_url_checks.py]: After talking with the autopush folks, I only need what is in test_urls.py

**Comment by rpappalax** [comment]
+1

**Pull request merged**